doc/testing.md

```diff
@@ -606,12 +606,14 @@ test/jdk/sun/security/pkcs11/README.
 
 ### Client UI Tests
 
+#### System key shortcuts
+
 Some Client UI tests use key sequences which may be reserved by the operating
 system. Usually that causes the test failure. So it is highly recommended to
 disable system key shortcuts prior testing. The steps to access and disable
 system key shortcuts for various platforms are provided below.
 
-#### MacOS
+##### MacOS
```

Comment:
```suggestion
##### macOS
```
Shall we spell _macOS_ with lower-case ‘m’ like Apple does?

Reply:
Done